src/django_components/component.py

```diff
@@ -103,12 +103,12 @@
 COMP_ONLY_FLAG = "only"
 
 # Define TypeVars for args and kwargs
-ArgsType = TypeVar("ArgsType", bound=tuple, contravariant=True)
-KwargsType = TypeVar("KwargsType", bound=Mapping[str, Any], contravariant=True)
-SlotsType = TypeVar("SlotsType", bound=Mapping[SlotName, SlotContent])
-DataType = TypeVar("DataType", bound=Mapping[str, Any], covariant=True)
-JsDataType = TypeVar("JsDataType", bound=Mapping[str, Any])
-CssDataType = TypeVar("CssDataType", bound=Mapping[str, Any])
+ArgsType = TypeVar("ArgsType", bound=tuple, contravariant=True, default=Any)
```

Comment:
This is what allows us to omit some of the types when configuring `Component[...]`